dashboard/test/models/peer_review_test.rb

```diff
@@ -67,6 +67,24 @@ def track_progress(level_source_id, user = @user)
     assert_equal PeerReview::REVIEWS_PER_SUBMISSION - 1, PeerReview.count - initial
   end
 
+  test 'instructor review should mark as accepted and delete outstanding unfilled reviews' do
+    level_source = create :level_source, data: 'My submitted answer'
+
+    Activity.create! user: @user, level: @script_level.level, test_result: Activity::UNREVIEWED_SUBMISSION_RESULT, level_source: level_source
+    track_progress level_source.id
+    review1 = PeerReview.offset(1).last
+    review2 = PeerReview.last
```

Comment:
They get reated as a part of track_progress in track_level_progress_sync in user.rb